static/scss/generic/_vertical-divider.scss

```diff
@@ -11,11 +11,11 @@
 ///     <div class="six-col">...</div>
 ///   <div class="six-col last-col">...</div>
 ///   </div>
-@mixin maas-vertical-divider {
+@mixin maas-dividers {
```

Comment:
This file should be renamed `_dividers.scss` or even `_patterns-dividers.scss` to be Vanilla.